search.html

```diff
@@ -286,6 +287,108 @@ <h2><a class="anchor" id="keyvalue"></a>Key-Value Pair Annotation Search</h2>
 
 <p>will return objects with a Key-Value pair annotation with the specific key: <span class="code-example">Temperature</span>.</p>
 
+<h2><a class="anchor" id="advanced"></a>Advanced Search</h2>
+
+<p>To understand basics of Search in OMERO, or to search for Key-Value pairs, please read also the <a href="http://help.openmicroscopy.org/search.html#tips"
+    target="_blank" class="external-url">Tips (wildcars),</a> <a href="http://help.openmicroscopy.org/search.html#keyword"
+    target="_blank" class="external-url">Keyword searching</a> and <a href="http://help.openmicroscopy.org/search.html#keyvalue"
+    target="_blank" class="external-url">Key-Value Pair Annotation Search</a> chapters. There is no difference in searching for Key-Value pairs between the basic and the <span class="bold">Advanced</span> search tabs.</p>
+
+<p>Advanced search is available in OMERO.web only.</p>
+
+<p>Inside the search page, click on the <span class="bold">Advanced</span> tab in the left-hand pane.</p>
+
+<p>OMERO.server uses <a href="https://lucene.apache.org/">Lucene</a> to perform the search commands. The <span class="bold">Advanced</span> tab accepts directly the Lucene queries, with no parsing. Follow the steps below to get started with Lucene queries.
+
+<p>Construct searches for a string in any <span class="bold">Field</span> as listed in <a href="https://docs.openmicroscopy.org/latest/omero/developers/Modules/Search.html#field-names"
+    target="_blank" class="external-url">the table of fields in the documentation</a> using the syntax: <span class="code-example">Field:string</span>.</p>
+
+<table>
+ <tr>
+  <th>Field</th>
+  <th>Comment</th>
+ </tr>
+ <tr>
+  <td>name</td>
+  <td>name of the field</td>
+ </tr>
+ <tr>
+  <td>details.owner.omeName</td>
+  <td>login name of the owner</td>
+ </tr>
+ <tr>
+  <td>details.owner.firstName</td>
+  <td>first name of the owner</td>
+ </tr>
+ <tr>
+  <td>details.owner.lastName</td>
+  <td>last name of the owner</td>
+ </tr>
+ <tr>
+  <td>details.group.name</td>
+  <td>group name of the group</td>
+ </tr>
+ <tr>
+  <td>tag</td>
+  <td>A Tag attached to the object</td>
```

Comment:
The descriptions seem to start getting initial capitals part way through, maybe best all lower.